### PR TITLE
Added 2 variables to flexibly control the background of the table cells

### PR DESCRIPTION
**Pull request by alxkosov**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 👻 What's the background?

1. Describe the source of requirement, like related issue link.
added 2 variables to flexibly control the background of the table cells when the row is selected and the sorting column is active
2. Describe the problem and the scenario.
the background of the cells of the selected row changed during the hover and for the cell of the column by which the sorting was performed

### 📝 Changelog

added two less-variable
| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [x ] Doc is updated/provided or not needed
- [ x] Demo is updated/provided or not needed
- [x ] TypeScript definition is updated/provided or not needed
- [x ] Changelog is provided or not needed

### CODE REVIEW

**Comment by zombieJ** [comment]
Hmmm. Why need diff selected with sort || hover color?

**Comment by alxkosov** [comment]
@zombieJ , thanks, 
added description to Pull Request

**Comment by alxkosov** [comment]
@zombieJ 
image

File: components/table/style/index.less [hunk 1]
```diff
@@ -233,6 +233,20 @@
         background: @table-row-hover-bg;
       }
     }
+    &.@{table-prefix-cls}-row-selected > td {
+      background: @table-selected-row-bg;
```
Comment (zombieJ) [inline components/table/style/index.less]: This is already defined and no need duplicate define: https://github.com/ant-design/ant-design/blob/master/components/table/style/index.less#L297

**Comment by zombieJ** [comment]
Got it. Could u help to change to feature branch?

File: components/table/style/index.less (hunk 1 above)
Comment (alxkosov) [reply components/table/style/index.less]: removed duplicate code

**Comment by alxkosov** [comment]
the pullrequest was moved to https://github.com/ant-design/ant-design/pull/16837

**Comment by alxkosov** [comment]
The Pull Request was moved to https://github.com/ant-design/ant-design/pull/16841

**Comment by alxkosov** [comment]
The Pull Request was moved to https://github.com/ant-design/ant-design/pull/16843

**Comment by zombieJ** [comment]
> The Pull Request was moved to #16843

👌

**Pull request closed without merging**